packages/cli/src/lib/certs/endpoints.ts

```diff
@@ -1,35 +0,0 @@
-import {APIClient} from '@heroku-cli/command'
-import * as Heroku from '@heroku-cli/schema'
-
-export function meta(appName: string, t: string, certName: string | undefined) {
-  let path
-  if (t === 'sni') {
-    path = `/apps/${appName}/sni-endpoints`
-  } else {
-    throw new Error('Unknown type ' + t)
-  }
-
-  if (certName) {
-    path = `${path}/${certName}`
-  }
-
-  return {path, flag: t}
-}
-
-function tagAndSort(appName: string, sniCerts: Heroku.SniEndpoint[]) {
-  sniCerts.forEach(function (cert) {
-    cert._meta = meta(appName, 'sni', cert.name)
-  })
-
-  return sniCerts.sort(function (a: Heroku.SniEndpoint, b: Heroku.SniEndpoint) {
-    const aName = a?.name || ''
-    const bName = b?.name || ''
-    return (aName > bName) ? 1 : ((bName > aName) ? -1 : 0)
-  })
-}
-
-export async function all(appName: string, heroku: APIClient): Promise<Heroku.SniEndpoint[]> {
-  const {body: sniCerts} = await heroku.get<Heroku.SniEndpoint[]>(`/apps/${appName}/sni-endpoints`)
-
-  return tagAndSort(appName, sniCerts)
-}
```

Comment:
This was completely removed because it was useless and replaceable inline for a fetch to the SNI Endpoints list endpoint in Platform API. Previously, SNI Endpoints were tagged with the type (SSL / SNI) and the path to fetch each one of those types, but now it was being always called with type 'sni' (SSL Endpoints no longer exist). Furthermore, they were being sorted by name, when this was uses only from `flags.ts` lib file and there, the SNI Endpoints list is filtered to find one single item (or throw errors), so it doesn't makes sense to sort it.